pkg/server/container_create_windows.go

```diff
@@ -517,18 +513,57 @@ func (c *criService) generateContainerSpec(id string, sandboxID string, sandboxP
 		}
 	}
 
+	// hyperv pci devices are exposed as windows devices on the container spec
```

Comment:
I find it unclear that this comment is talking specifically about hyper-v. Perhaps "For both LCOW and WCOW, devices can be passed through from the host to the container via the OCI Windows.Devices field"?